src/Form/atoms/SelectField.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useCallback, useState, SelectHTMLAttributes } from 'react';
+import styled, { css } from 'styled-components';
+import Label, { StyledLabel } from './Label';
+import { ISelectSizes } from '../index';
+import Icon, { IconWrapper } from '../../Icons/Icon';
+
+
+const Container = styled.div<{ styleSize?: ISelectSizes, }>`
+
+${({ styleSize }) => (styleSize === 'small') && css`
+  ${StyledLabel} {
+      span {
+        margin-bottom: 6px;
+      }
+    }
+  `}
+`;
+
+const StyledSelect = styled.select<{ styleSize?: ISelectSizes, activePlaceholder: boolean, selectWidth?: string }>`
+  box-sizing: border-box;
+  outline: none;
+  border-radius: 3px;
+  height: 40px;
+  padding: 0 25px 0 12px;
+  appearance: none;
+  width: ${({ selectWidth }) => selectWidth ? selectWidth : '220px'};
+
+  &::-ms-expand {
+    display: none;
+  }
+
+  ${({ theme }) => css`
+    border: 1px solid ${theme.styles.form.input.default.normal.borderColor};
+    font-family: ${theme.fontFamily.data};
+    ${theme.typography.form.input.value.normal};
+  `};
+
+  ${({ theme: { typography }, styleSize }) => (styleSize === 'small') && css`
+    height: 30px;
+    padding: 0 25px 0 5px;
+    ${typography.form.input.value.compact};
+  `};
+
+  ${({ theme, activePlaceholder }) => activePlaceholder && css`
+    ${theme.typography.form.input.placeholder.normal};
+  `};
+
+  ${({ theme, styleSize, activePlaceholder }) => (styleSize === 'small') && activePlaceholder && css`
+    ${theme.typography.form.input.placeholder.compact};
+  `};
+
+  option {
+    font-style: normal;
+    ${({ theme }) => css`
+      ${theme.typography.form.input.value.normal};
+    `};
+
+    ${({ theme: { typography }, styleSize }) => (styleSize === 'small') && css`
+      ${typography.form.input.value.compact};
+    `};
+  }
+`;
+
+const SelectWrapper = styled.div`
+  position: relative;
+  display: inline-block;
+
+  ${IconWrapper} {
+    position: absolute;
+    top: calc(50% - 7px);
+    right: 10px;
+  }
+`;
+
+interface ILabel {
+  htmlFor: string
+  labelText: string
+}
+
+interface OwnProps {
+  labelProps?: ILabel
+  styleSize?: ISelectSizes
```

Comment:
Simplify `styleSize` to `size` to stay consistent with how we're doing things in the `Button` component.

Reply:
sorry, I had a problem using size directly in the select because it's a property.
I have moved the value to the container and handled there.

Reply:
Sorry updating again.. so size in the select is the amount of rows allowed to be displayed.
I will keep the interface and the name to avoid confusion.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select

Reply:
Ahhhh, I didn't realise that size was a prop on select already. It's not something I've used much so it didn't occur to me.

Thinking ahead a little, the sizes we use for form fields should always ever be one of two sizes. The standard size (most commonly used) and the compact size. It's not like buttons where we have four sizes. So perhaps an option we can later integrate in to how we handle other fields would be to simply have this as a `boolean` value such as `isCompact`. What do you think?